### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by julianrogmans**
Ready for the first code review on create_ride_spec @seanculver

### CODE REVIEW

File: spec/features/create_ride_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+require "active_support"
```
Comment (websitewizardguy) [inline spec/features/create_ride_spec.rb]: Not sure if active support is necessary. Just forgot to remove it

File: spec/features/create_ride_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+feature 'User creates a ride' do
+  let(:origin) { Faker::Address.street_address }
+  let(:time) { Faker::Time.forward(2, :morning) }
```
Comment (seanculver) [inline spec/features/create_ride_spec.rb]: As a personal taste thing, I'd put a space between the `let` and the `scenario`

File: spec/factories/rides.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
```
Comment (seanculver) [inline spec/factories/rides.rb]: I might only include attributes that are required (not sure which are required and not), you can also use Faker or pass a block to get a more dynamic values if needed.

File: spec/controllers/rides_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (seanculver) [inline spec/controllers/rides_controller_spec.rb]: I might wait on committing this if there isn't a test that runs in this file.

**Comment by seanculver** [comment]
I'd format the description of the PR like such.

Creating the initial skeleton for Travel Rogue.
- Includes migrations for the rides model
- Test harness and simple test skeletons
- etc...

Describe what the pull request is and any important information in the PR.  Imagine it almost as a change log. It helps describe your intention of the changes, are you fixing something? are you building a feature? Are you working on a chore?

**Comment by seanculver** [comment]
It would be nice if you hooked up circleci to get the build status here. I don't see any views, is that feature test working?

**Pull request closed without merging**